### DISCUSSION TITLE
How do i change the xrdp login background

### DISCUSSION THREAD

**Discussion by Aerospace911** [neutrinolabs/xrdp#2746] [Q&A] [answered]
![image](https://github.com/neutrinolabs/xrdp/assets/79132854/bd1fef3e-46dc-4547-b4eb-36309392e0c3)
i converted my image to a bmp and put the full path but it doesnt work

**Comment by matt335672** [comment, marked as answer]
Have a look at the output of `xrdp -v`

If your version of xrdp is v0.9.18 or later, and xrdp has been built with `--with-imlib2`, you can use a jpg or png depending on your image, rather than a .bmp. You can also scale or zoom the image.

If xrdp hasn't been built with `--with-imlib2`, only a very limited subset of .bmp formats is supported - just enough to support the supplied bitmaps. You can't have alpha channels, colour spaces, gamma correction or ICC colour profiles. Looking at the source, compression isn't supported either. Bitmaps must be 4, 8, or 24 bit.

**Reply by Aerospace911** [reply]
do i need --with-imlib2 to zoom and scale the image or just a v0.9.18+ and if so, how do i scale the image?
ive tried ls_background_transform = scaled, ls_background_transform = scale, ls_background_transform = zoom, and ls_background_transform = zoomed, none of the above worked

**Reply by matt335672** [reply]
The scaling should work provided you're using xrdp >= 0.9.18. It's not supported for older versions.

Use `ls_background_transform=scale`.
